Services/User/classes/class.ilUserQuery.php

```diff
@@ -379,7 +379,7 @@ public function query()
             $where = " AND";
         }
         
-        if ($this->activation != "") {		// activation
+        if ($this->activation != "" && $this->activation != "both") {		// activation
```

Comment:
I think that and remove the 'both', as if you send in the 1 it should filter fine.